### DISCUSSION TITLE
Implementazione nuove funzionalità

### DISCUSSION THREAD

**Discussion by LinusJak** [rocksdanister/lively#1002] [General]
**Salve ed innanzi tutto complimenti per Lively**, che sto utilizzando con enorme piacere dopo aver abbandonato altre applicazioni simili, ma non sicuramente all'altezza di Lively. Probabilmente quello che sto per chiedere esiste già, ma non sono riuscito a trovarla o non lo so usare bene. C'è la possibilità che i video i salva schermi e tutte le clip che sono all'interno di Lively, possano essere attivate in modo sequenziale oppure random, con magari impostazioni di tempo prima di passare in automatico ad un nuovo Wallpaper video. Grazie ancora, spero di leggerVi in merito. Lino

**Comment by rocksdanister** [comment]
Currently under development https://github.com/rocksdanister/lively/issues/137
